### PR TITLE
update instructions to add a blueprint

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Partially closes https://github.com/camunda/developer-experience/issues/398.

Applying `hold` label as this should not be merged until https://github.com/camunda/web-modeler/pull/11106 is merged.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/launch-web-modeler.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To launch Web Modeler, follow the steps below:
 2. Select **Create new project** to create a new project and store diagrams.
    ![web modeler empty home](img/web-modeler-new-user-home.png)
 3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.
-4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
+4. Select **Browse blueprints** to open the blueprints dialog and view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use blueprint**, which will download the blueprint into the project and open it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
```
Comment (mesellings) [inline docs/components/modeler/web-modeler/launch-web-modeler.md:17]:
```suggestion
4. Select **Browse blueprints** to open the blueprints dialog and browse blueprints for various use cases as a starting point for your first diagram. Open a blueprint by selecting **Use blueprint**, which downloads the blueprint into the project and opens it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
```
Rewording suggestion? Also, possibly too much in this single step, could use a sub-bulleted list?

**Review by mesellings — DISMISSED** [review]
Lgtm 👍 (just a non-blocking rewording/list suggestion, feel free to ignore).

**Review by YanaSegal — DISMISSED** [review]
LGTM

File: docs/components/modeler/web-modeler/launch-web-modeler.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/modeler/web-modeler/launch-web-modeler.md:17]: I'll apply your changes and move a few things around -- also realized I still need to update the screenshot, so we can keep this on standby for a short while.

**Comment by christinaausley** [comment]
This is ready for final review 👍 Updated the screenshot and moved a bit of content around.

**Review by YanaSegal — APPROVED** [review]
LGTM :)

**Comment by akeller** [comment]
@christinaausley this looks ready to merge, can you confirm?

**Comment by christinaausley** [comment]
> @christinaausley this looks ready to merge, can you confirm?

There has not been a Web Modeler release with this UI update (but the PR is merged in so that's good!), so I've been hanging on to it. I can always backport if need be.

**Comment by christinaausley** [comment]
This is now live in the product -- thank you @ev-codes!

**Pull request merged**